Gemfile.lock

```diff
@@ -242,6 +244,7 @@ DEPENDENCIES
   cucumber-rails
   database_cleaner
   factory_girl_rails
+  font-awesome-sass (~> 4.4.0)
```

Comment:
My vote is to avoid adding Font Awesome for now. Let's limit the scope of this PR and replace icons in a later changeset.